client/src/components/layouts/global.css

```diff
@@ -65,15 +65,6 @@ body {
   min-height: 0;
 }
 
-#content-start {
-  min-height: 0;
-  height: 100%;
-}
-
-#content-start > div {
-  height: 100%;
-}
```

Comment:
Since we're fixing a regression, the priority should be to get a quick fix in and separately address any other issues that come up while fixing it.

That way there's no need to discuss the impact of other changes in the fix PR. We can do that later, knowing that the bug has been dealt with.

What I recommend is just deleting the offending css and creating another PR with the content-start changes so we can discuss/QA at our leisure.

Reply:
deleting the offending CSS, break the editor layout.

Reply:
I get what you mean, unfortunately, the CSS file affects the global CSS, and to decouple it, we need to restructure the architecture of how the files are handled.

Our choices are really limited 😞, I mentioned in the meeting, I think it's best to delete the page, because that is actually the simplest solution and re-arrange the structure can be done in another time, in another PR.

Reply:
Right, but we can do a small patch to the css to address this specific issue. I'll add a suggestion.